ome_model/experimental.py

```diff
@@ -167,7 +174,7 @@ class Plate(object):
 
     ID = 0
 
-    def __init__(self, name, rows, columns):
+    def __init__(self, name):
```

Comment:
That one is likely on me as this was introduced in https://github.com/ome/ome-model/pull/92 but never implemented. 

I suspect in the back of my mind, I was thinking of the `Plate.Rows` and `Plate.Columns` fields (see https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Plate) but never got to implement them.

Rather than deleting the arguments which is a breaking change (as you are updating the test), thoughts about making them optional e.g. `rows=None, columns=None`) and optionally filling the fields?